packages/simcore-sdk/src/simcore_sdk/node_ports_common/file_io_utils.py

```diff
@@ -47,6 +49,50 @@ class UploadableFileObject:
     file_size: int
 
 
+class ExtendedClientResponseError(ClientResponseError):
+    def __init__(
+        self,
+        request_info: RequestInfo,
+        history: tuple[ClientResponse, ...],
+        body: str,
+        *,
+        code: Optional[int] = None,
+        status: Optional[int] = None,
+        message: str = "",
+        headers: Optional[LooseHeaders] = None,
+    ):
+        super().__init__(
+            request_info,
+            history,
+            code=code,
+            status=status,
+            message=message,
+            headers=headers,
+        )
+        self.body = body
```

Comment:
MINOR: can you please add the links in the description here? I think it justifies better why would you want to expose the body in an error response